notebooks/functions/urban_population.py

```diff
@@ -0,0 +1,22 @@
+import pandas as pd
+import geopandas as gpd
+
+from functions.load_hexbins import load_hexbins
+from functions.urban_areas import select_urban_areas
+
+def urban_population(country_name:str, project):
+    
+    hexbins = load_hexbins(project)[['hex_id', 'country_subdivision', 'geometry', 'population']].\
```

Comment:
This is a significant simplification that depends on parameters (the size of the HexBin), so it would be MUCH better to use raw_population instead